examples/log_to_file/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "json_logger"
+version = "0.1.0"
+authors = ["Calvin Brown <cab025@protonmail.com"]
+edition = "2018"
+
+[dependencies]
+log = "0.4.8"
+log4rs = {version = "0.8.3", features = ["json_encoder"]}
```

Comment:
Should this be a path to the root? How do other projects do this?

Reply:
Good point, @estk https://github.com/SergioBenitez/Rocket/blob/v0.4/examples/json/Cargo.toml#L8 rocket looks like there is a path to the root. I'll fix this in both examples.